src/graphql/main/schema.graphql

```diff
@@ -322,6 +327,11 @@ input LnInvoiceCreateInput {
   """
   amount: SatAmount!
 
+  """
+  Optional callback for the lightning invoice updates.
```

Comment:
I don't think the callback should be at the invoice level. I think it should be at the account level. 

there are some reasons for that:
- the first one is that if the invoice is set at the invoice level, it will be easier for an attacker to have many urls as callback and trigger all of them with 1 sats invoices. 
- if we implement the callback at the account level, we can rate limit that, so that the callback url can only be changed once a day or something like that

The company/dev I talked to who wants to have callback functionality, they would want to have all the invoices coming back as a callback. so having to add callback for invoices one by one seems unecessary. 

the callback itself should have the information necessary for them to process it in the payload, instead of relying on a different callback url for each invoice.